src/views/Examples/Examples.js

```diff
@@ -28,11 +28,11 @@ class Examples extends Component {
             <h3>Checkbox</h3>
               <Checkbox>Check</Checkbox>
             <h3>DeleteLink</h3>
-              <DeleteLink url="/examples">Link</DeleteLink>
+              <DeleteLink id = "123" url="/examples">Link</DeleteLink>
```

Comment:
for delete and edit link it should be similar to this
url="/examples/delete/:id"

Reply:
i saw your latest changes. did we have an `id` prop at our components EditLink, DeleteLink? i think we should have it, in order to generate right url.